maintenance_timesheet/views/maintenance_request_views.xml

```diff
@@ -55,19 +55,13 @@
     <record id="hr_equipment_request_view_form" model="ir.ui.view">
         <field name="model">maintenance.request</field>
         <field name="inherit_id" ref="maintenance.hr_equipment_request_view_form" />
```

Comment:
Maybe you should use the view where the button box is inserted:

https://github.com/OCA/maintenance/blob/11.0/base_maintenance/views/maintenance_request_views.xml#L6

Reply:
OK, I'll fix it.

Neither v13 implementation (https://github.com/OCA/maintenance/blob/13.0/maintenance_timesheet/views/maintenance_request_views.xml#L77) uses base_maintenance view inheritance, so it's also incorrect, isn't it?